patch_api/api/orders_api.py

```diff
@@ -46,8 +46,8 @@ class OrdersApi(object):
         "average_daily_balance_btc_sats",
         "average_daily_balance_eth_gwei",
         "timestamp",
-        "origin_airport",
-        "destination_airport",
+        "origin_aiport",
+        "destination_aiport",
```

Comment:
Wait, is this a misspelling? Where is that coming from?

Reply:
That's what I'm trying to figure out 🤷 . I only made changes to issued_to and ran the generator.